resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/features/LocalDirsFeatureStep.scala

```diff
@@ -45,6 +47,10 @@ private[spark] class LocalDirsFeatureStep(
         new VolumeBuilder()
           .withName(s"spark-local-dir-${index + 1}")
           .withNewEmptyDir()
+            .withMedium(useLocalDirTmpFs match {
+              case true => "Memory" // Use tmpfs
```

Comment:
Think we shouldn't use `case true... false` - can instead do this:

```
.withNewEmptyDir().withMedium(if (useLocalDirTmpFs) "Memory" else null)...
```

Reply:
Done